src/relay/pass/pattern_util.h

```diff
@@ -119,6 +121,19 @@ inline bool IsDepthwiseConv2D(const Call& call,
       is_const_int(wshape[1], 1);
 }
 
+/*!
+ * \brief Get super-dimension of output channels of conv2d
+ * \param call The conv2d call.
+ * \return Super-dimension size of output channels of conv2d.
+ */
+inline int64_t GetConv2DSuperChannelsDim(const CallNode* call) {
```

Comment:
What is the "super dimension"?

Reply:
https://github.com/dmlc/tvm/blob/7a01476a3b27e4535767123d525a34f02f69dcdc/nnvm/include/nnvm/layout.h#L6
it simply means dimension denoted with upper case letter